smithy-model/src/main/java/software/amazon/smithy/model/traits/ExamplesTrait.java

```diff
@@ -337,6 +337,23 @@ public Builder content(ObjectNode content) {
                 return this;
             }
         }
+
+        @Override
+        public int hashCode() {
+            return Objects.hash(shapeId, content);
+        }
+
+        @Override
+        public boolean equals(Object other) {
+            if (other == this) {
+                return true;
+            }
+            if (other == null || other.getClass() != this.getClass()) {
+                return false;
+            }
+            ErrorExample otherExample = (ErrorExample) other;
+            return shapeId.equals(otherExample.shapeId) && content.equals(otherExample.content);
```

Comment:
just for extra safety: can we switch this to `Objects.equals`?

```suggestion
            return Objects.equals(shapeId, otherExample.shapeId) && Objects.equals(content, otherExample.content);
```